app/components/resource-sidebar/template.hbs

```diff
@@ -0,0 +1,48 @@
+<div class="sidebar-header">
+  {{#link-to 'stack' stack class="back-to-stack"}}
+    <i class="fa fa-chevron-left"></i>
+    <h3>{{resource.stack.handle}}</h3>
+  {{/link-to}}
+</div>
+
+<ul class="nav sidebar-nav">
+  <li class="title">
+    Apps
+    {{#aptible-ability scope="manage" user=user stack=stack as |hasAbility|}}
+      {{#if hasAbility}}
+        {{#link-to 'apps.new' stack }}
+          <i class="fa fa-plus"></i>
+        {{/link-to}}
+      {{/if}}
+    {{/aptible-ability}}
+  </li>
+
+  {{#each stack.apps as |app|}}
+    {{#link-to 'app' app tagName="li"}}
+      {{link-to app.handle 'app' app bubbles=false}}
+    {{/link-to}}
+  {{/each}}
+</ul>
+
+<ul class="nav sidebar-nav">
+  <li class="title">
+    Databases
+
+    {{#aptible-ability scope="manage" user=user stack=stack as |hasAbility|}}
```

Comment:
`aptible-ability` changed in 95c02b60e290cda4be36f66e43def160ae7341eb to have a `permittable` param instead of stack, so this should be:
`{{#aptible-ability scope="manage" user=user permittable=stack as |hasAbility|}}`